pkg/kafka/config/consumer/config.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2018 The Jaeger Authors.
```

Comment:
mb, I meant `pkg/kafka/config/consumer.go` and `pkg/kafka/config/producer.go`

Reply:
actually, keep as is, I prefer `producer.Configuration` over `config.ProducerConfiguration`

Reply:
Sounds good. I actually did it the first way, then noticed that it would be inconsistent with ES and Cassandra and changed it.

Reply:
I don't think we need the path of this file to be `pkg/kafka/config/consumer/config.go`; having this be `pkg/kafka/consumer/config.go` conveys the same information

Reply:
I'll make the change once the producer PR #957 is merged